pages/learn/manage/organizations.md

```diff
@@ -0,0 +1,97 @@
+
+* **Member**: Members are provided with read-only access to the organization with explicit access to its applications and teams. Roles on the application level are assigned by administrators. Members have no visibility over other applications and teams which they haven't been given access to in the organization.
+
+
+### Inviting members to an organization
+
+Now, with your organization created let's invite some members. Only the administrator can invite users to an organization. To invite, click the `Manage members` button on the organization summary dashboard. On the page that opens, click the `Invite member` button to open the `Invite organization member` modal. Here, you can either add users with their email or with their balena username. After assigning them roles in your organization. Click `Add member` to invite the user to your organization.
+
+<img alt="Invite Organization Member Modal" src="/img/common/app/" width="100%">
+
+On basis of the role assigned, they will be able to access, see, and manage both apps and other members inside the organization. To change the roles of a member, navigate to the `Organization Members` tab in the sidebar and change the `Organization role` of the desired user by using the dropdown list against their username.
+
+<img alt="Change Organization Member Roles" src="/img/common/app/" width="100%">
+
+
+## Creating teams in your organization
+
+With organizations, you can group your existing members to form teams as well. Teams are a robust way to manage members per their function and bulk provide access to applications as needed. Only administrators are allowed to create a team. To form a team, navigate to `Organization Teams` in the sidebar of the organization summary page. On the page that opens, click the `Create Team` button and enter the name of the team that is being created.
+
+<img alt="Create Team in Organization" src="/img/common/app/" width="100%">
+
+When the team is created, you can start adding members and applications to that specific team through the `Manage members` and `Manage applications` buttons respectively.
+
+<img alt="Organization Team Summary" src="/img/common/app/" width="100%">
+
+
+## Creating new applications under your organization
+
+With members invited to the organization, let's get started on creating new applications under an organization. Navigate to the `Organization Applications` tab in the sidebar to find the `Create application` button for creating a new application under your organization. Learn more about creating an application from the [getting started](getting-started) guide.
+
+<img alt="New Application under Organization" src="/img/common/app/" width="100%">
+
+### Moving existing applications to an organization
+
+Moving an existing application from your personal or any other organization to another organization is a common task that can be accomplished by [transferring ownership](transferring-ownership) of your application.
```

Comment:
`from your personal or any other organization to another organization`
→ `to an organization`
I.e.,
`Moving an existing application to an organization is a common task ...`